### ISSUE TITLE
FCOS + Cityscapes -> insufficient accuracy

### ISSUE THREAD

**Issue by BayMaxBHL** [open-mmlab/mmdetection#8364]
###  Question:
I use the Cityscapes dataset for testing target detection. Using the same training strategy (1 gpu) and dataset, the models are FasterRCNN, FCOS.
The FasterRCNN replication results are consistent with the official map, which is 42.
However, when using FCOS training, you can only get about 30.
The results of the two training models on COCO are approximately the same, but their performance on Cityscapes is quite different.
###  Seek help:
For Cityscapes datasets, can the authorities provide training results for more models?
Because COCO is relatively large, Cityscapes can provide faster reference in the early stages.

**Comment by BayMaxBHL** [comment]
FasterRCNN+Cityscapes:
![Faster-RCNN-2048](https://user-images.githubusercontent.com/34785944/178785416-ad09ed01-0253-4545-b34a-58935994d1be.png)

**Comment by BayMaxBHL** [comment]
FCOS + Cityscapes
![image](https://user-images.githubusercontent.com/34785944/178785930-c93ffa5d-1cb8-4e0e-b36e-a8d0e8c7bf2b.png)

**Comment by BayMaxBHL** [comment]
Fcos model-cfg is fcos_r50_caffe_fpn_gn-head_1x_coco.py . Number of categories changed to 8.

**Comment by BayMaxBHL** [comment]
Later, I tried to change the batch size, the learning rate configuration, and the stripe of fcos head, but there was no improvement.

**Comment by BayMaxBHL** [comment]
Later, I tried to use cloud services (2 GPUs), use the official configuration of fasterrcnn of cityscapes, and only replace the model part for training. ATSS quickly reached 36+, while FCOS finally reached less than 30.
![image](https://user-images.githubusercontent.com/34785944/179127399-46cc0223-3554-4f23-8104-4ddd4a2ccd2d.png)

**Comment by BayMaxBHL** [comment]
**I sincerely hope that mmdetection can provide a baseline for target detection of cityscapes without using the pre training model. Publicize the results of multiple models.**

**Comment by ZwwWayne** [comment]
Some parameters of FCOS and ATSS might need to be specifically tuned for Cityscapes. It is common to fine-tune the model for cityscapes using pre-trained models from COCO.
